examples/postgresql/README.md

```diff
@@ -0,0 +1,109 @@
+# User authentication in Postgresql using SVIDs with DNS names
+
+This is an example of how **spiffe-helper** can be used to authenticate users to a **PostgreSQL** database using x509 SVIDs.
+
+Postgres provides a feature that allows users to authenticate using [certificate authentication](https://www.postgresql.org/docs/9.5/auth-methods.html#AUTH-CERT). The server validates that the CN (Common Name) attribute of the certificate presented by the client matches the database user name or a [mapped](https://www.postgresql.org/docs/9.5/auth-username-maps.html) value.
+
+This guide sets up the authentication configuration for the username: `postgres-user`.
+
+
+## Guide
+
+### Considerations
+The following assumptions are made:
++ At least one SPIRE server and one agent are deployed with trust domain `example.org`.
+
++ Postgres 12 is used in this guide but each step should be easily applicable to other versions as well.
```

Comment:
for version < 10 there was not a way to reaload certificates from files, maybe you can add a note about that, for those version users must run something like:
```
# In postgres 9.5 only way to reload certificates is to restart it
/etc/init.d/postgresql restart

```